.github/workflows/labeler-promote.yml

```diff
@@ -0,0 +1,36 @@
+# Workflow template imported from:
+# https://github.com/dotnet/.github/workflow-templates
+#
+# Promote a model from staging to 'ACTIVE', backing up the currently 'ACTIVE' model
+name: "Labeler: Promotion"
+
+on:
+  # Dispatched via the Actions UI, promotes the model from a staged slot into the active prediction environment
+  workflow_dispatch:
+    inputs:
+      issues:
+        description: "Issues: Promote Model"
+        type: boolean
+        required: true
+      staged_key:
+        description: "The cache key suffix to use for promoting a staged model to 'ACTIVE'. Defaults to 'staged'."
+        required: true
+        default: "staged"
+      backup_key:
+        description: "The cache key suffix to use for backing up the currently active model. Defaults to 'backup'."
+        default: "backup"
+
+permissions:
+  actions: write
+
+jobs:
+  promote-issues:
```

Comment:
>promote-issues: [](http://example.com/codeflow?start=2&length=15)

We don't need to have similar check here:

github.repository_owner == 'dotnet'?

Reply:
Oh, looks this specific to the model if I understand it correctly

Reply:
Since the workflow is only triggered manually, we don't need to check that. We put the guard in place for workflows that run automatically (so that forks don't get red workflows).